calculator.c

```diff
@@ -0,0 +1,141 @@
+#include <stdio.h>
```

Comment:
Use Proper and uniform indentations in the file: Example: 
if () {
} else {
}
Use indentation plugins like Prettier, etc.

Reply:
I have updated the code formatting using Clang-Format. I use Clang-Format because Prettier doesn't support c/c++.